x/mongo/driver/topology/topology.go

```diff
@@ -122,6 +123,9 @@ func New(opts ...Option) (*Topology, error) {
 		dnsResolver:       dns.DefaultResolver,
 	}
 	t.desc.Store(description.Topology{})
+	t.updateCallback = func(desc description.Server) description.Server {
```

Comment:
Also a convenience change. Some tests in `topology_test.go` were declaring this exact function but it was duplicated because those tests only call `New` and not `Connect`. Putting it here means we don't have to individually update such tests if the function definition changes again in the future.